### PR TITLE
Dustin_L - Lesson03 - String Format Lab

### PR DESCRIPTION
**Pull request by rdlaner**
Addressed review comments on List Lab pull request.
Completed String Format Lab.

### CODE REVIEW

File: students/Dustin_L/lesson03/strformat_lab.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    data = [['Bert',     33, '-$150,000'],\
+            ['Angelina', 37, '-$1,500,000.42'],\
+            ['Batman',   1,  '$10,000'],\
+            ['Lucy',     5,  '$35,500.99']]
+
+    max_name = len(max([name[0] for name in data], key=len))
+    max_cost = len(max([cost[2] for cost in data], key=len))
+    max_age  = 3
+
+    str_fmt = '{:<' + f'{max_name + 5}' + '}{:<' + f'{max_age + 5}' +\
```
[the remark this answers is not in the document]
Comment (natasha-aleksandrova) [reply students/Dustin_L/lesson03/strformat_lab.py]: I'm assuming due to curly braces?

this should work (double the curly braces):
`str_fmt = f"{{:<{max_name + 5}}}{{:<{max_age + 5}}}{{:<{max_cost + 5}}}"`

[the remark this answers is not in the document]
Comment (rdlaner) [reply students/Dustin_L/lesson03/strformat_lab.py]: > Nice, that worked. I'm not really sure I understand why the extra set of curly braces is needed, though.

[the remark this answers is not in the document]
Comment (natasha-aleksandrova) [reply students/Dustin_L/lesson03/strformat_lab.py]: take a look at this doc (second paragraph):
https://docs.python.org/3/library/string.html#formatstrings

it's similar concept to escape characters and doubling up escape char when you actually want it to be literal
for example, if you're using `\` as escape char, then to use literal back slash you would do `\\`

[the remark this answers is not in the document]
Comment (rdlaner) [reply students/Dustin_L/lesson03/strformat_lab.py]: I see, thanks!

**Pull request merged**